docs/reference/configuration.md

```diff
@@ -47,6 +47,10 @@ This reference uses `.` to denote the nesting of values.
 * `blockstore.s3.credentials.access_key_id` `(string : )` - If specified, will be used as a static set of credential
 * `blockstore.s3.credentials.access_secret_key` `(string : )` - If specified, will be used as a static set of credential
 * `blockstore.s3.credentials.session_token` `(string : )` - If specified, will be used as a static session token
+* `blockstore.s3.endpoint` `(string : )` - If specified, custom endpoint for the AWS S3 API (https://s3_compatible_service_endpoint:port)
+* `blockstore.s3.force_path_style` `(boolean : false)` - When true, use path-style S3 URLs (https://<host>/<bucket> instead of https://<bucket>.<host>)
+    #force_path_style: true
```

Comment:
what is this comment?

Reply:
a mistake - copy-paste from my local configuration